### ISSUE TITLE
Missing constant in calculation of A and A_w?

### ISSUE THREAD

**Issue by tjdcs** [nschloe/note-on-cam16#1]
Hi Nico, 

In the change to a matrix-vector calculation it seems that the constant 0.305 is left out. The original equation reads 

```
(2 * R + G  + B/20 - 0.305)*Nbb
```

Is this a mistake in your manuscript or is the difference made up someplace else? I can't seem to find either an explanation or an obvious substitution. 

Thank you very much for the time, happy new year. 

-Tucker

**Comment by nschloe** [comment]
Where exactly?

**Comment by tjdcs** [comment]
In step 4* you replace the A and Aw equations with the first line of the matrix vector equation. A useful speedup but I am wondering about a constant that is located in the original equation. 

(step 6 from CAM16 by Li et. al, doi: 10.1002/col.22131)

![Screen Shot 2021-01-02 at 2 21 44 PM](https://user-images.githubusercontent.com/6669558/103464964-e52c6800-4d05-11eb-8c1b-65f3ddc218ad.png)

However, after testing your method as an implementation of CAM02 and CAM16 it is correct with the examples from "Color Appearance Modeling" Fairchild 2013 and Colorimetry 4th ed from CIE. 


TLDR what I thought was a bug doesn't appear to be, but I'm not sure where the difference is being made up.

**Comment by nschloe** [comment]
I see. In CAM16, they're adding `+0.1` to each of the RGB values. `2*0.1 + 0.1 + 0.1/20 = 0.305`, and it cancels out exactly. Not sure why they add it in the first place, but then again, the original algorithm is flawed in various places.

**Comment by tjdcs** [comment]
Ok. Thank you for the help. :) Much appreciated and thank you for your algorithm. It's about... 15-20% faster IIRC on my computer implemented in MATLAB
